techniques/failures/F7.html

```diff
@@ -1,8 +1,6 @@
-<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head>
+<!DOCTYPE html><html lang="ja" xml:lang="ja" xmlns="http://www.w3.org/1999/xhtml"><head>
 <meta charset="UTF-8">
-<title>F7: Failure of Success Criterion 2.2.2 due to an object or applet
-                    that has blinking content without a mechanism to pause the content that blinks
-                    for more than five seconds | WAI | W3C</title>
+<title>F7: 達成基準 2.2.2 の失敗例 － 5 秒よりも長く点滅するコンテンツを一時停止するメカニズムなしでコンテンツを点滅しているアプレット | WAI | W3C</title>
```

Comment:
原文では object or applet となっているので「オブジェクト又は」を追記しました。
```suggestion
<title>F7: 達成基準 2.2.2 の失敗例 － 5 秒よりも長く点滅するコンテンツを一時停止するメカニズムなしでコンテンツを点滅しているオブジェクト又はアプレット | WAI | W3C</title>
```